pkg/filter/filter_test.go

```diff
@@ -39,13 +39,16 @@ func (s *filterSuite) TestShouldUseDefaultRules(c *check.C) {
 	c.Assert(filter.ShouldIgnoreTable("metric_schema", "query_duration"), check.IsFalse)
 	c.Assert(filter.ShouldIgnoreTable("sns", "user"), check.IsFalse)
 	c.Assert(filter.ShouldIgnoreTable("tidb_cdc", "repl_mark_a_a"), check.IsFalse)
+	c.Assert(filter.ShouldIgnoreTable("mysql", "user"), check.IsFalse)
+	c.Assert(filter.ShouldIgnoreTable("mysql", "global_grants"), check.IsFalse)
+	c.Assert(filter.ShouldIgnoreTable("mysql", "tidb"), check.IsTrue)
 }
 
 func (s *filterSuite) TestShouldUseCustomRules(c *check.C) {
 	defer testleak.AfterTest(c)()
 	filter, err := NewFilter(&config.ReplicaConfig{
 		Filter: &config.FilterConfig{
-			Rules: []string{"sns.*", "ecom.*", "!sns.log", "!ecom.test"},
+			Rules: []string{"sns.*", "ecom.*", "!sns.log", "!ecom.test", "mysql.*", "mysql.user", "!mysql.global_grants"},
```

Comment:
"mysql.user" does not take effect in test, since `mysql.*` is used

Reply:
Fixed. Thanks!